scrapyd_k8s/api.py

```diff
@@ -7,6 +7,7 @@
 from natsort import natsort_keygen, ns
 
 from .config import Config
+from .k8s_resource_watcher import ResourceWatcher
```

Comment:
reference to k8s-specific term `k8s_resource_watcher`, this file should use the configured backend and launchers instead

Reply:
It is required for conditional start of joblogs feature in the run method and this method does not rely on a specific launcher, that is why I can't think of a clean way how to remove it from the api.py. But I am open for suggestions if you see a better way.

Reply:
If the `ResourceWatcher` is not Kubernetes-specific, perhaps the `k8s_` could be dropped from the filename?

Reply:
I do agree that it shouldn't be here and we don't want to violate SOLID, I will think how to refactor this.

Reply:
Resolved as a part of PR #38